docs/json.md

```diff
@@ -179,6 +180,13 @@ To set the `default` for a component of the JSON Schema, the annotation
 "! $default 'value'
 ```
 This also ensures that only components whose value is not equal to a specific default value are serialized to the JSON data file.
+
+Default values can only specified if one of the following rules apply:
+
+1. The default value represents the initial value of the underlying data type (e.g., `0` for an interger or `space` for a character field).
+2. The field and all higher-level structures of the field are marked as `$required`.
```

Comment:
I'm just wondering what about tables? The field with the default must be `$requried` (as well as all structures of higher-level until the table type), but not the table, right? Because here, we have the kind of "null" that means being not an entry.
So I'm not sure if I'm right and if you excluded tables from being `$required` accidentally or intentionally.

Reply:
Yes, I agree that it is complex. But I wonder whether that's the reason we should document it (somewhere) so that we do not need to think about it again

Reply:
Fixed with https://github.com/SAP/abap-file-formats/pull/690/commits/7c8b44bfa56ac5ecdf1c90d64a7e9fb885025858